ext/RMagick/rmmain.c

```diff
@@ -1295,12 +1313,9 @@ Init_RMagick2(void)
     DEF_ENUM(PixelInterpolateMethod)
         ENUMERATOR(UndefinedInterpolatePixel)
         ENUMERATOR(AverageInterpolatePixel)
-        ENUMERATOR(BicubicInterpolatePixel)
```

Comment:
@dlemstra what became of these enumerators?

Reply:
That is a very good catch @mockdeep. `BicubicInterpolatePixel` has been deprecated and replaced with `CatromInterpolatePixel`. `FilterInterpolatePixel` has been removed and there is no replacement. And `NearestNeighborInterpolatePixel` has been renamed to `NearestInterpolatePixel`. And we should have probably caught this before making all the breaking changes but I think we can continue with the 4.0 release after these issues have been addressed. I just opened a new PR to resolve this: #768